generator/src/test/java/com/scottlogic/deg/generator/cucumber/After.feature

```diff
@@ -0,0 +1,323 @@
+#Generator expected to generate in ascending order where there are bracketing constraints
+Feature: User can specify that a temporal date is lower than, but not equal to, a specified threshold
+
+  Background:
+    Given the generation strategy is full
+    And there is a field foo
+    And foo is of type "temporal"
+
+#Related to #667 - Expected time increment is one millisecond
+@ignore
+Scenario: 'After' valid date is successful
+  Given foo is after 2018-09-01T00:00:00.000
+    And the generator can generate at most 4 rows
+  Then the following data should be generated:
+    | foo                     |
+    | 2018-09-01T00:00:00.001 |
```

Comment:
Are we able to get a successful test if we generate just one row? It would be nice to have at least one happy path scenario that is not ignored

Reply:
Done